src/widget/notificationscrollarea.cpp

```diff
@@ -66,12 +66,23 @@ void NotificationScrollArea::trackWidget(GenericChatroomWidget* widget)
     }
 }
 
-void NotificationScrollArea::updateTracking()
+/**
+ * @brief Delete notification bar to visible elements on scroll area
+ */
+void NotificationScrollArea::updateVisualTracking() {
```

Comment:
This method is only used in the two `connect()` above, would it be possible to adjust the `connect()` calls to directly call `updateTracking()`?